src/Components/Blazor/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -0,0 +1,319 @@
+import { UserManager, UserManagerSettings, User } from 'oidc-client'
+
+type Writeable<T> = { -readonly [P in keyof T]: T[P] };
```

Comment:
Wowzers... 😨 

TypeScript provides very sophisticated ways for a dev to say what you can do with a certain type, and even more sophisticated ways of telling it to ignore what the first dev said!